src/openms/source/SYSTEM/File.cpp

```diff
@@ -229,6 +229,12 @@ namespace OpenMS
     //add data dir in OpenMS data path
     directories.push_back(getOpenMSDataPath());
 
+    //add root dir
+    directories.push_back('/');
```

Comment:
for absolute path I would suggest

```
       if (exists(filename))
       {
         return String(QDir::cleanPath(filename.toQString()));
       }
```

at the top of the fxn, e.g. line 222 - that sould basically return correctly for absolute path and the rest of the function is unchanged. Any reason why that would not work?